### PR TITLE
docs: replace later with breejs/later

### LINKED ISSUE #9826: Docs: Replace `later` with `@breejs/later`
We are using `@breejs/later` instead of `later` as `later`is no longer maintained. So we should update the docs where applicable and fix links to `@breejs/later` docs.

https://github.com/breejs/later

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Replace the `later` library with `@breejs/later` in the docs
- Drop runkit playground link
- Explain in some other sections as well that Renovate does not support using minutes or an exact time in the schedule

## Context:

Closes #9826.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/noise-reduction.md [hunk 1]
```diff
@@ -98,8 +98,8 @@ Or perhaps at least weekly:
   ]
 ```
 
-If you're wondering what is supported and not, under the hood, the schedule is parsed using [later.js](https://bunkat.github.io/later/) using the `later.parse.text(scheduleString)` API.
-[This page](https://bunkat.github.io/later/parsers.html#text) explains the supported syntax or you can experiment on the [RunKit playground](https://npm.runkit.com/later).
+If you're wondering what is supported and not, under the hood, the schedule is parsed using [@breejs/later](https://github.com/breejs/later) using the `later.parse.text(scheduleString)` API.
```
Comment (HonkingGoose) [inline docs/usage/noise-reduction.md:101]: Is this `later.parse.text(scheduleString)` API thing still valid now we use `@breejs/later`?

File: docs/usage/noise-reduction.md [hunk 2]
```diff
@@ -98,8 +98,8 @@ Or perhaps at least weekly:
...
+TODO INSERT PROPER LINK HERE explains the supported syntax DROPPED LINK TO RUNKIT PLAYGROUND.
```
Comment (HonkingGoose) [inline docs/usage/noise-reduction.md:102]: We used to have a link to some `parsers.html#text` docs, but I don't know what the replacement should be on the new https://breejs.github.io/later/getting-started.html docs. So if you code wizards can find out what the proper replacement is, that would be great! 😄 😉 

I dropped the link to the `runkit playground`, don't know if we need a replacement or not.

File: docs/usage/faq.md [hunk 3]
```diff
@@ -72,8 +72,8 @@ You can set a specific time zone in your local config file as well:
 The timezone must be a valid [IANA time zone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
 
 With the timezone set, you can define days of week or hours of the day in which Renovate will make changes.
-Renovate uses the [later](https://bunkat.github.io/later/parsers.html#text) library to parse the text.
-The _later_ library also handles the concepts of "days", time_before", and "time_after".
+Renovate uses the [@breejs/later](https://github.com/breejs/later) library to parse the text.
```
Comment (HonkingGoose) [inline docs/usage/faq.md:75]: Might also want to have a link to the parser docs for `@breejs/later` here as well.

File: docs/usage/configuration-options.md [hunk 4]
```diff
@@ -2116,7 +2116,8 @@ To restrict `aws-sdk` to only monthly updates, you could add this package rule:
 }
 ```
 
-Technical details: We mostly rely on the text parsing of the library [later](https://bunkat.github.io/later/parsers.html#text) but only its concepts of "days", "time_before", and "time_after" (Renovate does not support scheduled minutes or "at an exact time" granularity).
+Technical details: We mostly rely on the text parsing of the library [@breejs/later](https://github.com/breejs/later) but only its concepts of "days", "time_before", and "time_after".
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:2119]: Might also want to have a link to the parser docs for `@breejs/later` here as well.

File: docs/usage/faq.md (hunk 3 above)
Comment (viceice) [reply docs/usage/faq.md:75]: https://breejs.github.io/later/parsers.html#text

File: docs/usage/noise-reduction.md (hunk 1 above)
Comment (viceice) [reply docs/usage/noise-reduction.md:101]: sure, https://breejs.github.io/later/parsers.html#text

File: docs/usage/noise-reduction.md (hunk 2 above)
Comment (viceice) [reply docs/usage/noise-reduction.md:102]: https://breejs.github.io/later/parsers.html#text 🙃 

Maybe we should clarify that we don't support schedules which are smaller that hourly as renovate will only run hourly (or on hooks for app and on premise)

**Review by viceice — COMMENTED** [review]
😕  Did not count my review

**Review by viceice — APPROVED** [review]

**Pull request merged**